src/coreclr/gc/gcinterface.h

```diff
@@ -1025,6 +1025,7 @@ struct ScanContext
 {
     Thread* thread_under_crawl;
     int thread_number;
+    int thread_count;
```

Comment:
Will adding a field on `ScanContext` breaks the `GCtoEEInterface`?

Reply:
Probably. We may get lucky on 64-bit versions because `thread_count` would likely fall on an offset that was unused before (because both `thread_under_crawl` and `stack_limit` are 64-bit fields while `thread_number`and `thread_count`are both 32-bits. That would enable a new clrgc.dll running on an older coreclr.dll, but not the other way round.

Reply:
I checked and we are indeed fine on x64 / Windows as the offsets of the other fields didn't change with this change, for the reason explained above.